doc/source/math/discretization.rst

```diff
@@ -79,7 +81,8 @@ where :math:`I(\bar f; \cdot)` stands for the function
 Hence, this operator maps the finite array :math:`\bar f \in \mathbb{R}^n`
 to the abstract interpolating function :math:`I(\bar f; \cdot)`.
 
-This interpolation scheme is implemented in the `LinearInterpolation` operator.
+This interpolation scheme is implemented in the `LinearInterpolation`
+operator.
```

Comment:
Are these changes due to some auto-formatting thing in your text editor? I intentionally chose a line width of 100 to be able to write math formulas without breaking lines overly much.

If you could revert this, that would be great.